src/pages/signin/ValidateCodeForm/BaseValidateCodeForm.js

```diff
@@ -189,6 +189,7 @@ class BaseValidateCodeForm extends React.Component {
     }
 
     render() {
+        const hasError = Boolean(this.props.account) && !_.isEmpty(this.props.account.errors);
         return (
             <>
```

Comment:
Two Factor Authentication Missed here

Reply:
I couldn't find a way to pass the magic code. I'm changing the following line to return true, but it doesn't allow me to pass the magic code for the 2FA validation.

https://github.com/Expensify/App/blob/af70dc29746608efeb1c4c95c0a0dca731bfe122/src/libs/Permissions.js#L94

Reply:
I committed the solution method that I can't guarantee.

Reply:
That's behind beta, for I'll test and let you know!

Reply:
Thanks, I'm waiting